testing-a11y.md

```diff
@@ -0,0 +1,610 @@
+
+### Panelist projects leveraging this technique
+
+**Carbon Web Components**
+We can see a full investment into the outside-in pattern in Carbon Web Components' [`<bx-input>` element](https://github.com/carbon-design-system/carbon-web-components/blob/main/src/components/input/input.ts), including some additional slots for content beyond that covered herein.
+
+```html
+<bx-input>
+  <div slot="label-text">Label</div>
+  <div slot="helper-text">Description</div>
+</bx-input>
+```
+
+This allows the `<bx-input>` to fully leverage the accessibility relationship created by the outside-in pattern for the [`label-text` content](https://github.com/carbon-design-system/carbon-web-components/blob/aaaa893f15d1ab1dc08843fdf1261f02f249d6dd/src/components/input/input.ts#L212-L214). However, when taking a closer look, you'll see that the [`helper-text` and `validity-message` content](https://github.com/carbon-design-system/carbon-web-components/blob/aaaa893f15d1ab1dc08843fdf1261f02f249d6dd/src/components/input/input.ts#L233-L238) is not currently associated with the `<input>` element.
+
+**Spectrum Web Components**
+In order to attach description content to form elements, including the `<sp-textfield>` element in the Spectrum Web Components library, a `help-text` slot is surfaced. 
+
+```html
+<sp-textfield>
+  <div slot="help-text">Description</div>
+</sp-textfield>
+```
+
+This leverages the pattern outlined above very closely and expands to with a technique called [Stacked Slots](https://dev.to/westbrook/who-doesnt-love-some-s-3de0) that allows you to easily manage multiple pieces of description content based on the validity of the `<sp-textfield>` element. Even after all this time, I find that the patterns made available around slotted content and ensuring the accessibility of content leveraging shadow roots has much exploration to be had!
+
+**UI5 Web Components**
+In conjunction with a visual design decision that delivers extra content about the `<input>` element in a "popover", the `<ui5-input>` element from UI5 Web Components leverages a `valueStateMessage` slot similar to this pattern. *Notice that the `value-state` attribute must be set to `Information` for content supplied in this manner to be displayed.*
```

Comment:
*Notice that the `value-state` attribute must be set to `Information` for content supplied in this manner to be displayed.* 

Actually you could also set it to "Error", "Warning" and "Success". The value state message will display as long as the value-state property is set. What is more, if a value state is set and no value state message a default text depending on the value state is displayed in the popup and presented to the screen readers as well.

Reply:
@elenastoyanovaa I'm not seeing the `Success` value for this attribute allow a visitor to trigger the popover UI: https://webcomponents.dev/edit/J5evv2W65KHFCd1m5Q4r/src/index.ts?branch=ui5%40wfVpXdsTYhf9mY2slxW4t2Y2SjC3&p=stories

Thanks again for clarifying this usage!